packages/coinstac-client-core/src/model-service.js

```diff
@@ -81,7 +84,7 @@ class ModelService {
   /**
    * @abstract
    */
-  modelServiceHooks() {
+  static modelServiceHooks() {
```

Comment:
Bleh. This was super _gross_ to begin with, but making it `static` just makes it worse. Not sure it’s worth addressing in this PR.